java/arcs/core/storage/driver/volatiles/VolatileMemoryImpl.kt

```diff
@@ -44,6 +47,14 @@ class VolatileMemoryImpl : VolatileMemory {
     return@withLock currentEntry
   }
 
+  override fun countEntities(): Long {
+    return entries.values
+      .map { v -> v.data }
+      .filter { it is CrdtEntity || it is CrdtEntity.Data }
```

Comment:
Question for the reviewer: Does this cover all the entity cases (am I missing anything)?

Reply:
I think this is correct. If you want to be sure, you can write an integration test (eg use a handle to write some entities, then count them) - I think you are going to use this in an upcoming test so that may be enough to verify.